src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/groups/GroupsPage.java

```diff
@@ -119,16 +118,8 @@ public void removeUserGroupWithProjectLinks() {
 	}
 
 	public boolean notySuccessDisplayed() {
```

Comment:
Probably should change the method name since we are no longer using Noty

Reply:
Totally correct, updated in be07bdaee4bc923a93a456be164be77144a70e12, thanks